### PR TITLE
Remove extra click when creating dashboard cell

### LINKED ISSUE #3069: remove extra click when opening the CEO for the first time
the flow should be user adds a cell, clicks to add a graph, and when the ceo is opened for the first time, it should open to the query builder without requiring an extra click. 

remove this click

image

### PR DESCRIPTION
**Pull request by bthesorceror**
Closes #3069

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/dashboards/components/CellEditorOverlay.tsx [hunk 1]
```diff
@@ -185,203 +327,86 @@ class CellEditorOverlay extends Component {
...
-      editQueryStatus,
-    } = this.props
-
-    const {
-      activeQueryIndex,
-      isDisplayOptionsTabActive,
-      queriesWorkingDraft,
-      staticLegend,
-    } = this.state
+  private get source() {
```
Comment (121watts) [inline ui/src/dashboards/components/CellEditorOverlay.tsx]: Perhaps define the return type?

File: ui/src/dashboards/utils/sources.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const nextSource = (prevQuery, nextQuery) => {
```
Comment (121watts) [inline ui/src/dashboards/utils/sources.ts]: Perhaps define the return type?

File: ui/src/types/dashboard.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import {Query} from 'src/types'
```
Comment (121watts) [inline ui/src/types/dashboard.ts]: Love this

File: ui/src/dashboards/components/CellEditorOverlay.tsx (hunk 1 above)
Comment (bthesorceror) [reply ui/src/dashboards/components/CellEditorOverlay.tsx]: 👍

File: ui/src/dashboards/utils/sources.ts (hunk 2 above)
Comment (bthesorceror) [reply ui/src/dashboards/utils/sources.ts]: 👍

**Pull request merged**